### ISSUE TITLE
[BUG] Can not change mail account password after upgrade

### ISSUE THREAD

**Issue by changeme** [hestiacp/hestiacp#1962]
**Describe the bug**
After latest upgrade, cant change mail account password, on control panel or cli.

* On HestiaCP, there is no error on interfaz, so it's seems to be changed, but user cant login on webmail. Maybe it happens too on email clients like Outlook/Thunderbird (not tested)
* On cli, there is an error about libicuio missing, so its obvious that something went wrong

```/home/fakeuser/conf/mail/fakedomain/passwd``` file is changed, but there is not any password hash

**To Reproduce**

```
#v-change-mail-account-password fakeuser fakedomain fakemail fakenewpassword
/usr/local/hestia/php/bin/php: error while loading shared libraries: libicuio.so.60: cannot open shared object file: No such file or directory

#egrep fakemail /home/fakeuser/conf/mail/fakedomain/passwd
fakemail:{MD5}:fakeuser:mail::/home/fakeuser:50000:userdb_quota_rule=*:storage=50000M
```
**Expected behavior**
Change password

**Operating system:**

```
root@srv:~# cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
```

**Hestia Control Panel version:**

```1.4.5```

**Additional context**

Fix:

```
apt-get install libicu60
v-change-mail-account-password fakeuser fakedomain fakemail fakenewpassword
```

**Comment by jaapmarcus** [comment]
We have added: 

https://github.com/hestiacp/hestiacp/blob/b688e7b7213cb9b4aa8198cbb382f9faa436a8bd/src/hst_autocompile.sh#L491

But tested on multiple systems and haven't seen the issue before.

We will keep monitoring it ...

**Comment by mindaugasdi** [comment]
For me latest update installation

```
Ubuntu 18.04.5 LTS
Unpacking hestia (1.4.5) over (1.4.2) ...
Unpacking hestia-nginx (1.19.10) over (1.19.8) ...
Unpacking hestia-php (7.4.19) over (7.4.16) ..
```

failed to complete, because of similar reason: „error while loading shared libraries: libicuio.so.60“.

- Opening hestia resulted with 500 error.
- Rebooting did not help.
- All apt commands were failing with error:

```
liep. 04 19:30:01 [...] systemd[1]: Starting LSB: starts the hestia control panel...
liep. 04 19:30:01 [...] hestia[2207]:  * Starting hestia-nginx hestia-nginx
liep. 04 19:30:01 [...] hestia[2207]:    ...done.
liep. 04 19:30:01 [...] hestia[2207]:  * Starting hestia-php hestia-php
liep. 04 19:30:01 [...] hestia[2207]: /usr/local/hestia/php/sbin/hestia-php: error while loading shared libraries: libicuio.so.60: cannot open shared object file: No such file or directory
liep. 04 19:30:01 [...] systemd[1]: hestia.service: Control process exited, code=exited status=127
liep. 04 19:30:01 [...] systemd[1]: hestia.service: Failed with result 'exit-code'.
liep. 04 19:30:01 [...] systemd[1]: Failed to start LSB: starts the hestia control panel.
```

Writing `sudo apt-get install libicu60` solved the issue also.

**Comment by changeme** [comment]
I have same problem with HestiaCP.
Seems that service was down:

```
service hestia status
● hestia.service - LSB: starts the hestia control panel
   Loaded: loaded (/etc/init.d/hestia; generated)
   Active: failed (Result: exit-code) since Fri 2021-07-02 13:53:39 UTC; 2 days ago
     Docs: man:systemd-sysv-generator(8)
   CGroup: /system.slice/hestia.service
           ├─3209 nginx: master process /usr/local/hestia/nginx/sbin/hestia-nginx
           └─3210 nginx: worker process

jul 02 13:53:35 mysrv systemd[1]: Starting LSB: starts the hestia control panel...
jul 02 13:53:36 mysrv hestia[922]:  * Starting hestia-nginx hestia-nginx
jul 02 13:53:39 mysrv hestia[922]:    ...done.
jul 02 13:53:39 mysrv hestia[922]:  * Starting hestia-php hestia-php
jul 02 13:53:39 mysrv hestia[922]: /usr/local/hestia/php/sbin/hestia-php: error while loading shared libraries: libicuio.so.60: cannot open shared object file: No such file or directory
jul 02 13:53:39 mysrv systemd[1]: hestia.service: Control process exited, code=exited status=127
jul 02 13:53:39 mysrv systemd[1]: hestia.service: Failed with result 'exit-code'.
jul 02 13:53:39 mysrv systemd[1]: Failed to start LSB: starts the hestia control panel.
```

As I had installed libicuio.so.60, a `service hestia restart` fix it

Additional info:

```root@srv:~# v-list-sys-info
HOSTNAME             OS      VER    ARCH    HESTIA  RELEASE  UPTIME  LA
--------             --      ---    ----    ------  -------  ------  --
mysrv             Ubuntu  18.04  x86_64  1.4.5   release  14      0.01

```

```root@srv:~# v-list-sys-hestia-updates
PKG           VER      ARCH   UPDT  DATE
---           ---      ----   ----  ----
hestia        1.4.5    amd64  yes   2021-07-01
hestia-php    7.4.19   amd64  yes   2021-07-01
hestia-nginx  1.19.10  amd64  yes   2021-07-01

```

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/blob/2c05926619bdcc9b4f7e613a08a9649fdefba7dc/install/hst-install-ubuntu.sh#L34-L47

php7.4-intl is a currently installed on default and libicuio.so.xx is an dependency for it. It should be installed on default.

**Comment by MattFSGROUP** [comment]
> For me latest update installation
> 
> ```
> Ubuntu 18.04.5 LTS
> Unpacking hestia (1.4.5) over (1.4.2) ...
> Unpacking hestia-nginx (1.19.10) over (1.19.8) ...
> Unpacking hestia-php (7.4.19) over (7.4.16) ..
> ```
> 
> failed to complete, because of similar reason: „error while loading shared libraries: libicuio.so.60“.
> 
> * Opening hestia resulted with 500 error.
> * Rebooting did not help.
> * All apt commands were failing with error:
> 
> ```
> liep. 04 19:30:01 [...] systemd[1]: Starting LSB: starts the hestia control panel...
> liep. 04 19:30:01 [...] hestia[2207]:  * Starting hestia-nginx hestia-nginx
> liep. 04 19:30:01 [...] hestia[2207]:    ...done.
> liep. 04 19:30:01 [...] hestia[2207]:  * Starting hestia-php hestia-php
> liep. 04 19:30:01 [...] hestia[2207]: /usr/local/hestia/php/sbin/hestia-php: error while loading shared libraries: libicuio.so.60: cannot open shared object file: No such file or directory
> liep. 04 19:30:01 [...] systemd[1]: hestia.service: Control process exited, code=exited status=127
> liep. 04 19:30:01 [...] systemd[1]: hestia.service: Failed with result 'exit-code'.
> liep. 04 19:30:01 [...] systemd[1]: Failed to start LSB: starts the hestia control panel.
> ```
> 
> Writing `sudo apt-get install libicu60` solved the issue also.

Same here on both our nginx only servers. Command above sorted it! Cheers

**Comment by tlcd96** [comment]
Solving this, add the following name to the dependency list `libicu??` to get only the libicu60 or libicu64, etc...

Tested and works with apt and apt-get

**Comment by jaapmarcus** [comment]
Issue has been patched with the release of hestia-php-7.4.21
